Server/Game/tasks.py

```diff
@@ -4,122 +4,123 @@
 logger.addHandler(fh)
 logger.setLevel(logging.DEBUG)
 
-@celery.decorators.periodic_task(run_every=datetime.timedelta(seconds=5), mock_queue=None, mock_version=None, mock_game_users=None)
-def processMatchmakingQueue(mock_queue=None, mock_version=None, mock_maps=None, mock_game_users=None):
+@celery.decorators.periodic_task(run_every=datetime.timedelta(seconds=config.GAME_QUEUE_PROCESS_INTERVAL))
+def processMatchmakingQueue():
+    """
+    This is the entry point for the matchmaking logic. Celery workers will periodically execute
+    this logic and match players that were placed in the game queue. All the data needed to setup a game
+    will happen here.
 
-    logger.debug("Processing matchmaking queue")
+    The logging output for the matchmaking logic will be stored in matchmaking.log
+    """
+    try:
+        logger.debug("Processing matchmaking queue")
 
-    # Get the matchmaking queue
-    if not mock_queue:
+        # Get the matchmaking queue
         queue = Game_Queue.objects.filter()
-    else:
-        queue = mock_queue
 
-    # Get the latest version to use for creating games
-    if not mock_version:
+        # Get the latest version to use for creating games
         version = Version.objects.latest('pk')
-    else:
-        version = mock_version
 
-    # If there isn't at least 2 players in the queue then exit
-    if len(queue) < 2:
-        logger.debug("Not enough people in the queue to match...Exiting")
-        return
+        # If there aren't at least 2 players in the queue then exit
+        if len(queue) < 2:
+            logger.debug("Not enough people in the queue to match...Exiting")
+            return
 
-    # Take the first player of the queue
-    first_player = queue[0]
+        # Take the first player of the queue
+        first_player = queue[0]
 
-    # Randomly choose a second player from the remaining players on the queue
-    index = random.randint(1, len(queue) - 1)
-    second_player = queue[index]
+        # Randomly choose a second player from the remaining players on the queue
+        index = random.randint(1, len(queue) - 1)
+        second_player = queue[index]
 
-    logger.info("Found a match: " + str(first_player.user.username) + " vs. " + str(second_player.user.username))
+        logger.info("Found a match: " + str(first_player.user.username) + " vs. " + str(second_player.user.username))
 
-    # Determine the number of times these 2 players have played against each other
-    count = 0
-    for first_player_game_user in Game_User.objects.filter(user=first_player.user):
-        game_played = first_player_game_user.game
+        # Determine the number of times these 2 players have played against each other
+        count = 0
+        for first_player_game_user in Game_User.objects.filter(user=first_player.user):
+            game_played = first_player_game_user.game
 
-        if not game_played:
-            continue
+            if not game_played:
```

Comment:
Possibly not a necessary change, but this can be removed by adding an "exclude" to the above query:
filter().exclude(game=None)

Not sure how far we want to go in the name of cleanup, but there may even be a simple (maybe not super simple) query which ends in a count() which could accomplish all of this